lang/talon/talon-code_common_function.talon-list

```diff
@@ -0,0 +1,8 @@
+list: user.code_common_function
+tag: user.talon
+tag: user.talon_python
+-
+insert: insert
+key: key
+print: print
+repeat: repeat
```

Comment:
This is a useful example of how we might move all code_common_function lists to the new .talon-list format.